lib/puppet/face/help.rb

```diff
@@ -66,7 +66,7 @@
       facename, actionname = args
       if legacy_applications.include? facename then
         if actionname then
-          raise ArgumentError, "Legacy subcommands don't take actions"
+          raise ArgumentError, _("Legacy subcommands don't take actions")
```

Comment:
IIRC, contractions ("don't") pose challenges for i18n insofar as non-native speakers often lack the language context to properly expand the contraction. Maybe use "do not" in this case?